src/elements/OcAccordion.vue

```diff
@@ -27,30 +27,28 @@ export default {
       return "multiple: " + this.multiple
     },
   },
+  mounted() {
+    this.$on("toggle", id => {
+      const collapseOthers = !this.multiple
```

Comment:
Codacy found an issue: ['collapseOthers' is assigned a value but never used.](https://app.codacy.com/gh/owncloud/owncloud-design-system/pullRequest?prid=6407398)